runbotics/runbotics-scheduler/src/scheduler-database/credential/dto/update-credential.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+import { createCredentialSchema } from './create-credential.dto';
+import { Reveal } from '#/utils/generic.types';
+
+export const updateCredentialSchema = createCredentialSchema.pick({
+    name: true,
+    description: true,
+});
+
+type PartialUpdateCredentialDto = z.infer<typeof updateCredentialSchema>;
+
+export type UpdateCredentialDto = Reveal<
+    PartialUpdateCredentialDto & Required<Pick<PartialUpdateCredentialDto, 'name'>>
+>;
```

Comment:
We agreed that there will be no option to update credential details once created. It can stay ofc for the future, but it won't be used atm.